Makefile

```diff
@@ -59,7 +59,7 @@ lint:
 
 .PHONY: markdown-lint
 markdown-lint:
-	markdownlint $(TOP_LEVEL)/**/*.md
+	markdownlint *.md **/*.md .github/**/*.md
```

Comment:
this local run seems to be supported only on mac. There is no easy way to install that on linux?